expression/builtin_other.go

```diff
@@ -85,6 +85,7 @@ type inFunctionClass struct {
 }
 
 func (c *inFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (sig builtinFunc, err error) {
+	args = c.validateConstArgByType(args)
 	if err := c.verifyArgs(args); err != nil {
```

Comment:
Can we merge it into the `verifyArgs`, the name of this two is very similar.

Reply:
updated.